eth/v1alpha1/beacon_chain.proto

```diff
@@ -273,6 +273,26 @@ message ValidatorChangeSet {
     repeated bytes public_keys = 2;
 }
 
+// Request for indexed attestations by target epoch.
+message ListIndexedAttestationsRequest {
+    oneof query_filter {
```

Comment:
Do you need this oneof? 

Given that the target_epoch is the only query field here, zero or unset should indicate genesis epoch.

Reply:
Discussed offline. The default behavior for unset criteria should be that we return the current epoch information.